.changeset/modern-parrots-talk.md

```diff
@@ -1,5 +0,0 @@
----
-'lit-html': patch
----
```

Comment:
I think this needed to also include `lit`.

Reply:
or maybe we can try adding this option for `lit` and the core packages?
https://github.com/changesets/changesets/blob/main/docs/config-file-options.md#fixed-array-of-arrays-of-package-names

Reply:
Hmm, sounds like `fixed` is symmetrical, so that would mean if `lit` gets bumped, `lit-html` would always get bumped too. We only want it to be in one direction.

Reply:
Added to lit